simulariumio/file_converter.py

```diff
@@ -19,24 +21,207 @@
 class FileConverter(TrajectoryConverter):
     def __init__(self, input_file: InputFileData):
         """
-        This object loads the data in .simularium JSON format
-        at the input file
+        This object loads data from the input file in .simularium format.
```

Comment:
Since you moved the writing code out into a `writers` section, why not move the reading code into a `readers` section?